ext/com_dotnet/com_typeinfo.c

```diff
@@ -282,9 +282,7 @@ PHP_COM_DOTNET_API ITypeLib *php_com_load_typelib_via_cache(const char *search_s
 		goto php_com_load_typelib_via_cache_return;
 	}
 
-	name_dup = estrndup(ZSTR_VAL(key), ZSTR_LEN(key));
```

Comment:
The reason this does a copy is that strtok modifies the input string.

Reply:
Should I duplicate it here, or within php_com_load_typelib()?

Reply:
The duplication in php_com_load_typelib() makes sense to me -- it's an implementation detail.